src/server/info_collector.cpp

```diff
@@ -228,6 +228,8 @@ info_collector::app_stat_counters *info_collector::get_app_counters(const std::s
     INIT_COUNTER(check_and_mutate_bytes);
     INIT_COUNTER(read_bytes);
     INIT_COUNTER(write_bytes);
+    INIT_COUNTER(compaction_write_rate_megabytes);
```

Comment:
Better to use bytes instead of MB for our metrics. `*1024` if rocksdb returns MB.